### ISSUE TITLE
Hyper-parameter of fine-tune

### ISSUE THREAD

**Issue by zhi-xuan-chen** [Ziyan-Huang/STU-Net#9]
I am trying to reproduce your fine-tune results on the Amos 2022 dataset using STU-Net_small. What is the best hyper-parameter setting for this model and dataset. Much appreciated if you can provide!

**Comment by Ziyan-Huang** [comment]
In addition to the default nnU-Net settings, we have only adjusted the learning rate to be 10 times smaller. However, further tuning of the learning rate and the number of training epochs may potentially yield better results. Please follow the instructions in the README to reproduce the results, and feel free to experiment with these parameters. If you have any questions or need assistance, don't hesitate to ask.

**Comment by zhi-xuan-chen** [comment]
ok, thank you for your help！
